lib/facebook/messenger/server.rb

```diff
@@ -14,9 +14,8 @@ def call(env)
         @request = Rack::Request.new(env)
         @response = Rack::Response.new
 
-        case
-        when @request.get? then verify
-        when @request.post? then receive
+        if @request.get? then verify
+        elsif @request.post? then receive
```

Comment:
I prefer `case` to `elsif` in general, but especially if there's a chance there might be more conditions down the road. I couldn't find anything on this in [bbatsov/ruby-style-guide](https://github.com/bbatsov/ruby-style-guide), though – what do you think, @hyperoslo/web?